tests/go/GofmtBearTest.py

```diff
@@ -5,3 +5,11 @@
     GofmtBear,
```

Comment:
Thanks. I thought that the variable should be named `GofmtBearTest` but since it was *working code* and this was my first PR, I didn't dare touch it. 😆